lib/elixir/src/elixir_exp.erl

```diff
@@ -313,6 +313,8 @@ expand({Name, Meta, Kind} = Var, #{vars := Vars} = E) when is_atom(Name), is_ato
           compile_error(Meta, ?m(E, file), "expected var \"~ts\"~ts to expand to an existing variable "
                         "or be part of a match", [Name, elixir_scope:context_info(Kind)]);
         true ->
+          Message = io_lib:format("variable ~ts is used as function call", [Name]),
```

Comment:
We need to add to the error message: ", please use parentheses to remove the ambiguity and solve this warning"